modules/features2d/src/sift.simd.hpp.orig

```diff
@@ -0,0 +1,865 @@
+// This file is part of OpenCV project.
```

Comment:
I suppose that the file was added to repo by mistake.

Reply:
Removed in the latest PR, apologies.